build/teamcity/cockroach/nightlies/pebble_nightly_metamorphic_race_impl.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+dir="$(dirname $(dirname $(dirname $(dirname "${0}"))))"
+source "$dir/teamcity-bazel-support.sh"  # For process_test_json
+
+set -euxo pipefail
+ARTIFACTS_DIR=/artifacts/meta
+mkdir -p $ARTIFACTS_DIR
+GO_TEST_JSON_OUTPUT_FILE=/artifacts/test.json.txt
+
+echo "TC_SERVER_URL is $TC_SERVER_URL"
+
+bazel build //pkg/cmd/bazci //pkg/cmd/github-post //pkg/cmd/testfilter --config=ci
+
+BAZEL_BIN=$(bazel info bazel-bin --config race --config ci)
```

Comment:
The `bazel info` invocation should match up to what you have in `bazel build`. In this case it's `--config ci`, not `--config race --config ci`.